src/main/java/com/flagsmith/FlagsmithClient.java

```diff
@@ -398,7 +416,32 @@ public Builder setDefaultFlagPredicate(@NonNull Predicate<String> defaultFlagPre
      */
     public Builder setDefaultFlagValueFunction(
         @NonNull Function<String, String> defaultFlagValueFunction) {
-      this.client.defaultFlagValueFunc = defaultFlagValueFunction;
+      this.configuration.flagsmithFlagDefaults.setDefaultFlagValueFunc(defaultFlagValueFunction);
+      return this;
+    }
+
+    /**
+     * When a flag does not exist in Flagsmith or there is an error, the SDK will return an empty
+     * list of flags by default. For example:
+     * - if you call identifyUserWithTraits(...) and the call fails
```

Comment:
Maybe this is my lack of understanding of the javadoc formatting but prepending these lines with a '-' makes it seem like it's some form of bulleted list, can we remove these and just add an indent? Or perhaps you can point me somewhere that defines this as a standard in javadoc?

Reply:
p.s. sorry, I know this one is quite picky 😬 !

Reply:
Added proper javadoc list

Reply:
Really sorry @sw-tracker, my comment was not necessarily regarding the syntax of the list, but the fact that it's a list in the first place - it doesn't feel like a list? Those 2 items should flow as a single sentence right? Or am I missing something?

Reply:
Switch to paragraph style explanation